src/core/thread/mesh_forwarder.cpp

```diff
@@ -610,38 +600,44 @@ otError MeshForwarder::HandleFrameRequest(Mac::Frame &aFrame)
     return error;
 }
 
-otError MeshForwarder::SendFragment(Message &aMessage, Mac::Frame &aFrame)
+void MeshForwarder::SendFragment(Message &aMessage, Mac::Frame &aFrame)
```

Comment:
Given that there is only one call to this, should we just call `PrepareDataFrame()` directly in `HandleFrameRequest()`?

Reply:
Good point. Will change it.